xwiki-platform-core/xwiki-platform-url/xwiki-platform-url-default/src/main/java/org/xwiki/url/internal/DefaultURLConfiguration.java

```diff
@@ -60,4 +63,16 @@ public boolean useResourceLastModificationDate()
     {
         return this.configuration.get().getProperty(PREFIX + "useResourceLastModificationDate", true);
     }
+
+    @Override
+    public List<String> getTrustedDomains()
+    {
+        return this.configuration.get().getProperty(PREFIX + "trustedDomains", Collections.emptyList());
+    }
+
+    @Override
+    public boolean skipTrustedDomainsChecks()
```

Comment:
The name of the method sounds weird, I would expect it to skip the trusted domain check and not indicate if it's skipped. What about something like isTrustedDomainsCheckSkipped (not really sure what "checks" was plural too) ?

Reply:
yeah I don't like the name too much too. I'd make it shorter and remove "check" which doesn't mean much IMO, and instead talk about activating/enabling the feature or not, with something like `enableTrustedDomains` or `isTrustedDomainsEnabled`. WDYT?

Reply:
Ok to go with `isTrustedDomainsEnabled` but then the default would be true.